components/InputField/styles.ts

```diff
@@ -21,20 +22,12 @@ const styles = StyleSheet.create({
   },
 
   section: {
-    display: "flex",
-    flexDirection: "column",
+    alignItems: "center",
+    flexDirection: "row",
+    flex: 1,
     justifyContent: "center",
+    maxHeight: 70,
```

Comment:
Since the input's height is fixed, there should be no need to specify `maxHeight` for the section itself

Reply:
It's because I need the flex: 1 parameter so the bar fills the maximum amount of space horizontally, but I don't want it to take space vertically. That's the way I found to do it, do you think I should do it another way ?

Reply:
oh, i see. I would use the `width: "100%"` for such a component, but using flex + bounded height works as well!